lib/sms_autofill.dart

```diff
@@ -8,9 +8,9 @@ import 'package:pin_input_text_field/pin_input_text_field.dart';
 export 'package:pin_input_text_field/pin_input_text_field.dart';
 
 class SmsAutoFill {
-  static SmsAutoFill _singleton;
+  static SmsAutoFill? _singleton;
   static const MethodChannel _channel = const MethodChannel('sms_autofill');
-  final StreamController<String> _code = StreamController.broadcast();
+  final StreamController<String?> _code = StreamController.broadcast();
```

Comment:
not really sure we push null values inside the stream, if we do we probably shouldn't I'll check more in details later as it's in native side